### PR TITLE
gifs, text, style

### PR DESCRIPTION
**Pull request by diminator**
Added text and some more images
Some final checks are required:
- [x] content re-read + (css)style check
- [x] add README, LICENSE and CONTRIBUTING
- [x] deploy on webservice
- [x] serve under bigchaindb.com (either tutorials.bigchaindb.com/crab or bigchaindb.com/tutorials/crab)
- [ ] link, from bigchaindb.com, blogpost, tweet and other marketing stuff

### CODE REVIEW

**Comment by jernejpregelj** [comment]
Added README, LICENSE, CONTRIBUTING
Added docker but I need to take another look into client setup

**Comment by jernejpregelj** [comment]
webpack works from docker now

**Comment by kremalicious** [comment]
just leaving this here for documentation reasons:

- we've setup tutorials.bigchaindb.com
- which is just a CNAME and points to a S3 bucket
- for now, we just deploy new tutorials in a subfolder in that bucket, so for this project under `/crab`
- as soon as need arises, we move the S3 bucket contents onto a proper web server or service like Now/Heroku
- we might want to add an index page under tutorials.bigchaindb.com as soon as we have more than 2 or 3 tutorials. Handy for SEO too.

**Comment by jernejpregelj** [comment]
https://tutorials.bigchaindb.com/crab/
It's up, Index and Error document in S3 bucket settings should both point to index.html

**Comment by kremalicious** [comment]
OMG, first time I'm seeing this in full, this is amazing! Awesome work! With real pixel crabs! Sorry I only got around checking this now.

Some suggestions, I'll leave that to you if you wanna do this as part of this PR or not:

## UX

The navbar is slightly confusing cause it's supposed to be a progress bar: fails to give sense of place when landing on page, provides no feel of accomplishment, and borked on mobile. Could be all solved by:

- adding Start as first menu item and give that the 0 index number (yes, I know, logo link, but that's not obvious for everyone)
- would introduce an incomplete/completed state, like so:

  screen shot 2017-09-05 at 20 46 32
  So by default they're dimmed, then a completed and an active style. Think we can skip the active underline here, looks too busy otherwise.
- Cause we're agile and all, UTF-8 is our friend for checkmark icon needs: [✓](http://www.fileformat.info/info/unicode/char/2713/browsertest.htm)
- use our scrolling overflow menu pattern: [CSS](https://github.com/ipdb/site/blob/master/_src/_assets/scss/_menus.scss#L76) | [HTML](https://github.com/ipdb/site/blob/master/_src/_includes/menu-main.html#L11). That's currently in use on bigchaindb.com & beta.ipdb.io

Install step:
This is confusing cause it looks like a button:
screen shot 2017-09-05 at 21 36 26
- makes definitely sense highlighting it more than a simple code block, so maybe make it like so with border and slightly more space:
screen shot 2017-09-05 at 21 39 43

Transaction tools:
- should be fullwidth, stacked layout on mobile, works quite well if code is just on top, then button, then response field
- should have some sort of state feedback during sending and prevent double sending. Could be as simple as disabling the button as soon as user hits it and switching the button text to "Sending...", "Crunching...", "Crabbing..." or something. Or just switch text for a spinner. That's on my list over on site too so whatever you choose here I'll probably just steal it for https://github.com/ascribe/bigchain-website/issues/135 :-)

Pages feel cutoff at the end, like on laptop sized viewports:
- as quick win, giving them all a bit more bottom space would be enough. But even better, add a general footer which would solve this visual problem and additionally:

## Marketing

From marketing side it's important to think of this tutorial as a landing page. So we wanna funnel people through our web properties when they arrive on this tutorial page as their first touch point, or after finishing it. Having some links in a footer would help with that. As links maybe:

- [bigchaindb.com](https://www.bigchaindb.com)
- [Get Started](https://www.bigchaindb.com/getstarted/)
- [FAQ](https://www.bigchaindb.com/faq/)
- [Docs](https://docs.bigchaindb.com/)
- [Blog](https://blog.bigchaindb.com/)

Footer can be real simple, just `display: inline-block` links in a row with some space between. Maybe try a slightly darker background for whole thing or add a simple 1px border at top of footer.

Twitter Cards/Open Graph:
We need to spam the `head`with Twitter Cards & Open Graph markup if we wanna tweet/Facebook/Slack/LinkedIn about it in style. You can just copy them out of live bigchaindb.com and modify the values. Dynamic ones are [here](https://github.com/ascribe/bigchain-website/blob/master/_src/_includes/twittercards.html) & [here](https://github.com/ascribe/bigchain-website/blob/master/_src/_includes/opengraph.html) otherwise.

As sharing image use [this one](https://github.com/ascribe/bigchain-website/blob/master/_src/_assets/images/share-image-getstarted.png) or just point to its live version: https://www.bigchaindb.com/assets/img/share-image-getstarted.png

File: src/app/components/Burn.jsx [hunk 1]
```diff
@@ -30,37 +36,56 @@ class Burn extends TutorialStep {
             })
             .then((burnedCrab) => {
                 this.setState({
-                    output: JSON.stringify(burnedCrab.metadata, null, 2)
+                    output: JSON.stringify(
+                        burnedCrab.txHistory.reverse().map(tx => tx.data)
```
Comment (diminator) [inline src/app/components/Burn.jsx]: `tx.metadata`

**Comment by jernejpregelj** [comment]
Added footer, previous steps are checked and green, steps collapse on mobile

**Pull request merged**